podcasts/Onboarding/Models/PlusPurchaseModel.swift

```diff
@@ -119,7 +119,13 @@ private extension PlusPurchaseModel {
         guard let parentController else { return }
 
         let navigationController = parentController as? UINavigationController
-        let controller = WelcomeViewModel.make(in: navigationController, displayType: .plus)
+
+        let controller: UIViewController
+        if FeatureFlag.patron.enabled, SubscriptionHelper.activeSubscriptionTier == .patron {
```

Comment:
A small suggestion for readability improvement:

```suggestion
        if FeatureFlag.patron.enabled, SubscriptionHelper.activeTier == .patron {
```

Reply:
Done here: 14e98944ed09bc2c65d24e64a386c22ae67b73f9